interface-definitions/qos.xml.in

```diff
@@ -132,6 +132,12 @@
                       <valueless/>
                     </properties>
                   </leafNode>
+                  <leafNode name="triple-isolate">
+                    <properties>
+                      <help>Flows are defined by the 5-tuple, and fairness is applied over source and destination addresses intelligently (ie. not merely by host-pairs), and also over individual flows (default)</help>
```

Comment:
I'm not against including this option, but the description from the kernel docs doesn't make a good help string. We need to come up with a concise description of what it does — i.e., what the effect of enabling it is.

Reply:
Hey @dmbaturin, I totally understand where you're coming from. I think my goal here was to match the style of the existing help strings. It's probably going to be difficult to come up with a more concise string that still accurately describes how this flag works, because it does quite a lot in terms of flow isolation. How about something like this?

`Fairness is applied to individual flows (defined by the 5-tuple) as well as groups of flows with the same source and destination addresses (default)`

Reply:
In VyOS help string terms (default) indicates that it is enabled if not set on the CLI which is not the case.

Is this the default behavior of CAKE In the Kernel?

Reply:
Hi @c-po, yes, the default behaviour for CAKE in the kernel is to use the `triple-isolate` method of flow isolation. I'm not sure how long this has been the case, but I've seen it referenced as the default setting as far back as at least 2020. https://man7.org/linux/man-pages/man8/tc-cake.8.html

Reply:
We probably should have a way to remove it the from the config if the user does not want it?

Reply:
Hi @c-po, disabling flow isolation entirely requires passing the `flowblind` option when configuring the shaper. This is already available in the VyOS CLI as `blind`.

Reply:
@MattKobayashi thanks, that's what I remembered. Is there a specific reason we need this CLI option?

Reply:
Hi @c-po, I think completeness is the primary reason. Users who aren't intimately familiar with CAKE may assume that the `triple-isolate` simply isn't available in VyOS, without realising that it's actually the default.

It could simply be printed in the CLI menu for CAKE that the default is `triple-isolate` without offering an explicit setting, but I wanted to include it as an option in case someone wished to have it reflected in their configuration. Another option might be to make it clear in the VyOS documentation that `triple-isolate` is the default behaviour when no other option is configured. I note that there's no VyOS documentation for CAKE yet, I'd be happy to take on the task of writing this.

Reply:
Sorry for this to be stalling - I have not yet my head arround that how to properly add it to the CLI